documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -2756,12 +2776,25 @@ See the xref:{link-oracle-connector}#oracle-data-type-mappings[list of Oracle-sp
 |`0`
 |Specifies, in milliseconds, how frequently the connector sends messages to a heartbeat topic. +
 Use this property to determine whether the connector continues to receive change events from the source database. +
-It can also be useful to set the property in situations where the connector no change events occur in captured tables for an extended period. +
-In such a a case, although the connector continues to read the redo log, it emits no change event messages, so that the offset in the Kafka topic remains unchanged.
+It can also be useful to set the property in situations where no change events occur in captured tables for an extended period. +
+In such a case, although the connector continues to read the redo log, it emits no change event messages, so that the offset in the Kafka topic remains unchanged.
 Because the connector does not flush the latest system change number (SCN) that it read from the database, the database might retain the redo log files for longer than necessary.
 If the connector restarts, the extended retention period could result in the connector redundantly sending some change events. +
 The default value of `0` prevents the connector from sending any heartbeat messages.
 
+|[[oracle-property-heartbeat-action-query]]<<oracle-property-heartbeat-action-query, `+heartbeat.action.query+`>>
+|No default
+||Specifies a query that the connector executes on the source database when the connector sends a heartbeat message. +
```

Comment:
```suggestion
|Specifies a query that the connector executes on the source database when the connector sends a heartbeat message. +
```
There's an extra leading pipe character that's messing with the table formatting.